core/domain/feedback_jobs_continuous.py

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""Domains for continuous-computation."""
```

Comment:
I also don't understand what this means. What is a "domain"?

A better way to explain this is something like "Continuous computations relating to the feedback system.", since every class here is a subclass of a continuous computation manager. The concept of continuous jobs is explained in the wiki. In general your docstrings need to be understandable by contributors who are new to the codebase.

Reply:
Done.